sdl-core/src/main/scala/io/smartdatalake/app/GlobalConfig.scala

```diff
@@ -58,6 +58,13 @@ import org.apache.spark.util.PrivateAccessor
  * @param synchronousStreamingTriggerIntervalSec Trigger interval for synchronous actions in streaming mode in seconds (default = 60 seconds)
  *                       The synchronous actions of the DAG will be executed with this interval if possile.
  *                       Note that for asynchronous actions there are separate settings, e.g. SparkStreamingMode.triggerInterval.
+ * @param allowAsRecursiveInput allow exception for specific DataObjects to the validation rules for Action.recursiveInputIds.
```

Comment:
```suggestion
 * @param allowAsRecursiveInput Whitelist of DataObjects for which the validation rules for Action.recursiveInputIds are not checked.
```

Reply:
Also from what I understood we want to encourage people to first try Action.recursiveInputIds and only if that's not enough try allowAsRecursiveInput on DataObject side, right? Maybe we should guide them first to Action.recursiveInputIds either via Exception Message or via doc